Dockerfile

```diff
@@ -2,14 +2,16 @@
 FROM ubuntu:20.04 as builder 
 MAINTAINER Kevin Kreiser <kevinkreiser@gmail.com>
 
+ARG CONCURRENCY=$(nproc)
```

Comment:
the docker build killed my computer a few times, so I added this:

`docker build --build-arg CONCURRENCY=20 -t valhalla/valhalla:run-latest .`